packages/calendar-utils/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@commercetools-uikit/calendar-utils",
```

Comment:
Would it make sense to group such packages together? For example `calendar-utils` and `calendar-time-utils`

Reply:
my idea was to split these two because they rely on two different moment dependencies.

Reply:
I see, ok